translator/trace/protospan_translation.go

```diff
@@ -61,15 +61,3 @@ func StatusCodeFromHTTP(httpStatusCode int) pdata.StatusCode {
 	}
 	return pdata.StatusCodeError
 }
-
-// AttributeValueToString converts an OTLP AttributeValue object to its equivalent string representation
-// Deprecated: use pdata.AttributeValueToString instead.
-func AttributeValueToString(attr pdata.AttributeValue) string {
-	return pdata.AttributeValueToString(attr)
-}
-
-// AttributeMapToMap converts an OTLP AttributeMap to a standard go map
-// Deprecated: use pdata.AttributeMapToMap instead.
-func AttributeMapToMap(attrMap pdata.AttributeMap) map[string]interface{} {
-	return pdata.AttributeMapToMap(attrMap)
-}
```

Comment:
Probably should be marked as breaking change in the changelog?

Reply:
Not really... Maybe ? :))

Reply:
Done.